packages/core/src/sso/SamlConnector/index.ts

```diff
@@ -149,16 +154,15 @@ class SamlConnector {
 
       // eslint-disable-next-line new-cap
       const identityProvider = saml.IdentityProvider({
-        wantAuthnRequestsSigned: true, // Sign auth request by default
```

Comment:
@darcyYe PTAL, this is a bug per our offline discussion

Reply:
Fix in https://github.com/logto-io/logto/pull/4878.